README.md

```diff
@@ -36,14 +37,14 @@ The OpenWhisk CLI is written in the Go language. You have two options to build
 the binary locally:
 
 1.  Build using the packaged Gradle scripts (including the 'gogradle' plugin),
-now the preferred build method.
+    now the preferred build method.
```

Comment:
Move up a line?

Reply:
The line is more than 80 characters so most IDE's cut it off. I indented it so it would align with text above it:
![image](https://user-images.githubusercontent.com/4106/44436567-ebca4280-a583-11e8-9705-80f0d8ac5751.png)